### PR TITLE
[stable/insights-agent] bump nova version and add new params

### PR DESCRIPTION
**Pull request by vitorvezani**
**Why This PR?**
Bump nova version that has support for `--helm` and `--containers` flags for `find` command
Requires [Nova 3.4 release](https://github.com/FairwindsOps/nova/pull/154)

Fixes #

**Changes**
Changes proposed in this pull request:

* Bump version
* Add new flags to the find command

**Checklist:**

* [X] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [X] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/insights-agent/templates/nova/cronjob.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ spec:
             command:
               - /nova
               - find
+              - -- helm
```
Comment (vitorvezani) [inline stable/insights-agent/templates/nova/cronjob.yaml]: @sudermanjr, @rbren - should these flags be added via config file? Would the below work?
```
nova:
  ...
  config:
    context: ""
    desired-versions: {}
    include-all: true
    wide: true
    output-file: "/output/nova.json"
    helm: true
    containers: true
```

Comment (rbren) [reply stable/insights-agent/templates/nova/cronjob.yaml]: I'm not sure on this one...I think flags are fine though.

File: stable/insights-agent/templates/nova/cronjob.yaml [hunk 2]
```diff
@@ -27,6 +27,8 @@ spec:
...
+              - -- containers
```
Comment (rbren) [inline stable/insights-agent/templates/nova/cronjob.yaml]:
```suggestion
              - --helm
              - --containers
```

File: stable/insights-agent/templates/nova/cronjob.yaml (hunk 1 above)
Comment (rbren) [reply stable/insights-agent/templates/nova/cronjob.yaml]: Flags are probably better for these, since we don't really want people to override them.

Comment (vitorvezani) [reply stable/insights-agent/templates/nova/cronjob.yaml]: I wonder if `.Config.configLocation` will not override those flags

**Review by rbren — APPROVED** [review]

File: stable/insights-agent/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 2.6.11
+* Update `nova` version to `v3.4` 
+* Add new flags to `nova find` command
```
Comment (ivanfetch-wt) [inline stable/insights-agent/CHANGELOG.md]: An non-blocking nit: mentioning what the new flag is/does will help the reader know how nova may behave differently.

**Review by ivanfetch-wt — APPROVED** [review]

File: stable/insights-agent/CHANGELOG.md (hunk 3 above)
Comment (vitorvezani) [reply stable/insights-agent/CHANGELOG.md]: done

**Pull request merged**